tests/losses/sinkpit_wrapper_test.py

```diff
@@ -0,0 +1,150 @@
+from asteroid.losses import sdr
+from asteroid.losses import singlesrc_mse, pairwise_mse, multisrc_mse
+from asteroid.engine.system import System
+from asteroid.utils.test_utils import DummyWaveformDataset
+
+# target modules
+from asteroid.losses import SinkPITLossWrapper, SinkPITBetaScheduler, sinkpit_default_beta_schedule
+
+
+def bad_loss_func_ndim0(y_pred, y_true):
+    return torch.randn(1).mean()
+
+
+def bad_loss_func_ndim1(y_pred, y_true):
+    return torch.randn(1)
+
+
+def good_batch_loss_func(y_pred, y_true):
+    batch, *_ = y_true.shape
+    return torch.randn(batch)
+
+
+def good_pairwise_loss_func(y_pred, y_true):
+    batch, n_src, *_ = y_true.shape
+    return torch.randn(batch, n_src, n_src)
+
+
+@pytest.mark.parametrize("batch_size", [1, 2, 8])
+@pytest.mark.parametrize("n_src", [2, 5, 10])
+@pytest.mark.parametrize("time", [16000, 1221])
+def test_wrapper(batch_size, n_src, time):
+    targets = torch.randn(batch_size, n_src, time)
+    est_targets = torch.randn(batch_size, n_src, time)
+    for bad_loss_func in [bad_loss_func_ndim0, bad_loss_func_ndim1]:
+        loss = SinkPITLossWrapper(bad_loss_func)
+        with pytest.raises(AssertionError):
+            loss(est_targets, targets)
+
+    loss = SinkPITLossWrapper(good_pairwise_loss_func)
+    loss(est_targets, targets)
+    loss_value, reordered_est = loss(est_targets, targets, return_est=True)
+    assert reordered_est.shape == est_targets.shape
+
+
+@pytest.mark.parametrize("batch_size", [1, 2])
+@pytest.mark.parametrize("n_src", [2, 3, 4])
+@pytest.mark.parametrize("beta,n_iter", [(100.0, 2000)])
+@pytest.mark.parametrize(
+    "function_triplet",
+    [
+        [sdr.pairwise_neg_sisdr, sdr.singlesrc_neg_sisdr, sdr.multisrc_neg_sisdr],
+        [sdr.pairwise_neg_sdsdr, sdr.singlesrc_neg_sdsdr, sdr.multisrc_neg_sdsdr],
+        [sdr.pairwise_neg_snr, sdr.singlesrc_neg_snr, sdr.multisrc_neg_snr],
+        [pairwise_mse, singlesrc_mse, multisrc_mse],
+    ],
+)
+def test_proximity_sinkhorn_hungrian(batch_size, n_src, beta, n_iter, function_triplet):
+    time = 16000
+    noise_level = 0.1
+    pairwise, nosrc, nonpit = function_triplet
+
+    # random data
+    targets = torch.randn(batch_size, n_src, time) * 10  # ground truth
+    noise = torch.randn(batch_size, n_src, time) * noise_level
+    est_targets = (
+        targets[:, torch.randperm(n_src), :] + noise
+    )  # reorder channels, and add small noise
+
+    # initialize wrappers
+    loss_sinkhorn = SinkPITLossWrapper(pairwise, n_iter=n_iter)
+    loss_hungarian = PITLossWrapper(pairwise, pit_from="pw_mtx")
+
+    # compute loss by sinkhorn
+    loss_sinkhorn.beta = beta
+    mean_loss_sinkhorn = loss_hungarian(est_targets, targets, return_est=False)
+
+    # compute loss by hungarian
+    mean_loss_hungarian = loss_hungarian(est_targets, targets, return_est=False)
+
+    # compare
+    assert_allclose(mean_loss_sinkhorn, mean_loss_hungarian)
+
+
+class testCallback(pl.callbacks.Callback):
```

Comment:
This should have an uppercase T: `TestCallback`. I wonder why the linters don't complain about it.

Reply:
Also don't know. 
Useless linter ^^